lib/mix/lib/mix/tasks/xref.ex

```diff
@@ -72,25 +80,29 @@ defmodule Mix.Tasks.Xref do
   ## Unreachable
 
   defp unreachable(pair) do
+    excludes = excludes()
+
     for manifest <- E.manifests(),
         {_, sources} = parse_manifest(manifest),
         source(source: file) = source <- sources,
-        entries = unreachable_source(source),
+        entries = unreachable_source(source, excludes),
```

Comment:
Do we need to have `excludes` in a variable, or can we just call `excludes/0` here?